build-support/bin/native/bootstrap_rust.sh

```diff
@@ -48,7 +48,8 @@ function bootstrap_rust() {
     "${RUSTUP}" toolchain install ${RUST_TOOLCHAIN}
     "${RUSTUP}" component add --toolchain ${RUST_TOOLCHAIN} ${RUST_COMPONENTS[@]} >&2
 
-    ln -fs "$(RUSTUP_TOOLCHAIN="${RUST_TOOLCHAIN}" cargo_bin)" "${rust_toolchain_root}/${cargo_versioned}"
+    symlink_target="$(perl -e 'use File::Spec; print File::Spec->abs2rel(@ARGV) . "\n"' "$(RUSTUP_TOOLCHAIN="${RUST_TOOLCHAIN}" cargo_bin)" "${rust_toolchain_root}")"
```

Comment:
It seems a little wanton to add perl as a dep for developers for this even though perl is ~everywhere. Consider `python -c 'import os, sys; print(os.path.relpath(*sys.argv[1:]))'`.